Services/Devices.cs

```diff
@@ -242,6 +242,18 @@ public async Task<Device> CreateAsync(string deviceId)
                 var device = new Azure.Devices.Device(deviceId);
                 device = await this.registry.AddDeviceAsync(device);
 
+                // Use the same credentials as are used when we create a 
+                // device with bulk import.
+                device.Authentication = new AuthenticationMechanism
```

Comment:
this needs to be done before creating the device (line 243)

Reply:
Fixed.